src/community.rs

```diff
@@ -2,29 +2,46 @@ use near_sdk::borsh::{self, BorshDeserialize, BorshSerialize};
 use near_sdk::serde::{Deserialize, Serialize};
 use near_sdk::{env, AccountId};
 
+#[derive(BorshSerialize, BorshDeserialize, Serialize, Deserialize, Clone)]
+#[serde(crate = "near_sdk::serde")]
+pub struct WikiPage {
+    enabled: bool,
```

Comment:
enabled isn't needed. because     `pub wiki1: Option<WikiPage>` is an option, when it's None (in Near.call API, null), it means not enabled. Same for github board     `pub github: Option<String>,`

Reply:
Got it, thanks!